File: MenagerieMadness/Models/Question.cs
using System;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;

namespace MenagerieMadness.Models
{
    public class Question
    {
        public int id { get; set; }
        public int userId { get; set; }
        public string question { get; set; }

        public string answer { get; set; }

        public string wrong1 { get; set; }
        public string wrong2 { get; set; }

        


    }
}

File: MenagerieMadness/client/src/components/Question.js
import React from "react";
import { Card, CardImg, CardBody } from "reactstrap";

const Question = ({ question }) => {
  return (
<Card className="m-4">
<p className="text-left px-2">Posted by: {question.userId}</p>
<CardBody>
        <p>
          <strong>{question.question}</strong>
        </p>
        <p>{question.answer}</p>
        <p>{question.wrong1}</p>
        <p>{question.wrong2}</p>
      </CardBody>
    </Card>
  
  );
};

export default Question;